modal/cls.py

```diff
@@ -428,6 +428,24 @@ def _get_method_names(self) -> Collection[str]:
         # returns method names for a *local* class only for now (used by cli)
         return self._method_partials.keys()
 
+    @live_method
+    async def _get_constructor_args(self) -> typing.Sequence[api_pb2.ClassParameterSpec]:
```

Comment:
These are the `Cls` variants of schema getters which makes sense to have since you want this info before constructing any instance of the class (for building uis etc). It does clutter the namespace of the Cls though, for something that's quite internal at this point, so maybe the names should be a bit more mangled? `._modal_get_constructor_args()`?

Reply:
Good idea. It's a bit annoying though since it needs to hook into the lazy-loader functionality (the `live_method` thing), but I'll see if I can do that in a nice way for standalone functions too...

Reply:
Wasn't so bad. Just had to wrap the helper in a synchonize_api